README.md

```diff
@@ -514,11 +558,11 @@ To start the ballerina service using the configuration file, run the following c
 ```
    $ ballerina run restful_service/ --config restful_service/ballerina.conf
 ```
-NOTE: The above configuration is the minimum configuration needed to enable tracing and metrics. With these configurations default values are load as the other configuration parameters of metrics and tracing.
+>NOTE: The above configuration is the minimum configuration needed to enable tracing and metrics. With these configurations, default values are loaded for the rest of the configuration parameters of metrics and tracing.
 
 ### Tracing 
 
-You can monitor ballerina services using in built tracing capabilities of Ballerina. We'll use [Jaeger](https://github.com/jaegertracing/jaeger) as the distributed tracing system.
+You can monitor Ballerina services using built-in tracing capabilities of Ballerina. We'll use [Jaeger](https://github.com/jaegertracing/jaeger) as the distributed tracing system.
```

Comment:
Minor corrections:
"You can monitor Ballerina services using the built-in tracing capability of Ballerina. Let's use [Jaeger](https://github.com/jaegertracing/jaeger) as the distributed tracing system here.
Follow the steps below to use tracing with Ballerina."